ui/client/summary.js

```diff
@@ -53,9 +53,19 @@ const Summary = () => {
           console.log('Container auto-shutdown sequence initiated');
         });
     }
-    // if we arrive with the terminal param but no lock, get rid of the terminal param
+
+    // save an empty variable so we can clear the timeout later on
+    let clearLock;
```

Comment:
Try: checking inside the timeout if the conditions are met, then maybe we don't need to clear the timeout
Question: what's happening with the closure re clearlock? If we do it the current way, what about cleaning up the timeout?